### PR TITLE
Add special buttons to Polish keyboard

### LINKED ISSUE #52: Additional buttons don't appear on Polish keyboard layout
**Describe the bug**
While English(UK) keyboard works OK, Polish keyboard does not have additional buttons (Ctrl, Alt, Esc etc.).
This is the only way I could make them appear:
https://github.com/lukasaristide/improved-osk-gnome-ext/commit/681817e4194178cf205c7a33aed871898410f3f1
though I suppose this is not the desired way - since this is not how it's done for other keyboard layouts.

**Environment (those are the ones I verified):**
 - Distro: Fedora 38, Ubuntu 23.04, Ubuntu 22.10
 - Gnome version: 43, 44

### PR DESCRIPTION
**Pull request by lukasaristide**
This resolves #52

### CODE REVIEW

File: src/data/osk-layouts/pl.json [hunk 1]
```diff
@@ -457,41 +497,67 @@
...
-            "level": 2,
-            "width": 1.5
+            "level": 2
           },
           {
-            "action": "emoji",
-            "iconName": "face-smile-symbolic",
-            "width": 1.5
+            "keyval": "0xffeb",
+            "label": "Super"
```
Comment (nick-shmyrev) [inline src/data/osk-layouts/pl.json:525]: Same as above:
```suggestion
            "action": "modifier",
            "keyval": "0xffeb",
            "label": "Super"
```

File: src/data/osk-layouts/pl.json [hunk 2]
```diff
@@ -209,41 +216,67 @@
...
```
[the remark this answers is not in the document]
Comment (lukasaristide) [reply src/data/osk-layouts/pl.json:244]: This was intended - with this modifier one cannot use it as button itself (changing views), which is the most common use case for me. If You want to place it here as the modifier however (for compatibility with other languages), I can change it to what You suggested. Do You want me to?

[the remark this answers is not in the document]
Comment (lukasaristide) [reply src/data/osk-layouts/pl.json:244]: Actually, Super+A covers also its solo behavior, I'll commit this.

**Review by nick-shmyrev — APPROVED** [review]
Looking good, merging this in. Thanks for the contribution!

**Pull request merged**